uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -246,16 +252,80 @@ define([
     function openActiveSectionTab (tabIndex) {
       var activeSection = getActiveSection();
       var tabs = activeSection.tabs;
+      var nextTab = tabs[tabIndex];
 
       tabs.forEach(function (tab) {
         tab.active = false;
       });
 
-      tabs[tabIndex].active = true;
+      if (!nextTab) {
+        if (tabIndex === -1) {
+          openPreviousSection();
```

Comment:
Should not we call this function as `openPreviousTab` and `openNextTab`?

Reply:
No, let's say you have 3 tabs in a section, meaning their indexes are 0, 1 and 2. If you ask to open index `-1` that's not a previous tab - that's **previous section**. If you ask index 3, then it is **next section**.